admin/DumpGenresTxt

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env perl
+
+use warnings;
+use strict;
+
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+
+use Encode qw( encode );
+use File::Spec;
+use Getopt::Long;
+use MusicBrainz::Server::Context;
+
+my $show_help = 0;
+my $database = 'MAINTENANCE';
+my $output_dir = '.';
+my $compress = 0;
+
+GetOptions(
+    'help' => \$show_help,
+    'database=s' => \$database,
+    'output=s' => \$output_dir,
+    'compress!' => \$compress,
+) or exit 2;
+
+sub usage {
+    print <<~'EOF';
+        Usage: DumpGenresTxt [options]
+
+            Dumps a list of all genres names to genres.txt, separated
```

Comment:
Renaming the script seems fine, though I kinda liked the idea of hosting it at `https://musicbrainz.org/genres.txt` rather than `/genre-names.txt`, so that it corresponds with the existing https://musicbrainz.org/genres page by just appending .txt.  Of course, we don't have to serve the file as the same file name, but it seems nice to keep the name the same, so that when you grep `genres.txt` you can see where it comes from.